src/main.c

```diff
@@ -380,17 +380,45 @@ int main(int argc, char ** argv){
       if(opt.rank == 0)
         u_call_cmd("LANG=C free -m");
     }
-
+  
     MPI_Barrier(MPI_COMM_WORLD);
     if(opt.rank == 0){
       fprintf(file_out, "\n[%s]\n", phase->name);
+      
+      if(opt.pause_dir){
+        // if the file exists
+        char path[2048];
```

Comment:
linux/limits.h.
#define PATH_MAX        4096    /* # chars in a path name including nul */

#include <linux/limits.h>

char current_path[PATH_MAX];

Use this define rather than a hard-coded value for better portability and compatibility.

Reply:
Agreed that PATH_MAX might be better. The article here describes general challenges:
https://eklitzke.org/path-max-is-tricky
Since IOR uses PATH_MAX, I made changes accordingly.